#46 duo admin amendments - Changed variables names in actions | Chang… (#238)
rapid7/insightconnect-plugins: duo_admin/help.md

# Version History

> * 3.3.3 - Changed variables names in actions | Changed `Exception` to `PluginException` | Moved constants to class init | Use fstring instead concatenation

Reviewer: ```suggestion
* 3.3.3 - Changed `Exception` to `PluginException` | Moved constants to class init | Use fstring instead of concatenation
```
